### DISCUSSION TITLE
Add way to search for recipes without specific tags and categories / Tag and category exclusions

### DISCUSSION THREAD

**Discussion by BeatLink** [mealie-recipes/mealie#3396] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

It is difficult to find recipes that lack a certain tag or category

### Please provide a concise description of the feature that would resolve your issue.

A specific list filter or search mechanism, to locate recipes that lack a specific tag and recipes that lack a certain category. 

For example, there could be a switch in the tag / category filters to find recipes not matching the selected items

### Please consider and list out some caveats or tradeoffs made in your design decision

Slightly increased UI complexity

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by basings** [comment]
I like this idea. I came here looking for this since filtering on "no category" is not yet possible.

Being able to filter based on the absence of any (specific) category lets the user detect missing recipes.

**Comment by LeskoIam** [comment]
It would be very useful when bulk importing recipes and no categories are set. It would go hand in hand with random meal selection based on category...

**Reply by uberguy16** [reply]
This would be so helpful for us. Occasionally we forget to apply a category but going through every recipe to find them is tedious. The Tags show up from the recipe list but not the Categories.

**Comment by sir106** [comment]
looked for the same! Fully support the idea to search for receipes without tags or category
